src/App.svelte

```diff
@@ -40,12 +41,12 @@
       <Main
           on:leave-rally={() => {
             store.updatePlatformEnrollment(false);
-            // reset to first run until Rally uninstalls itself.
-            firstRun = true;
           }}
         />
     {/if}
   {/if}
 {:else}
-  <div>Sorry, Rally is not supported in this locale.</div>
+  <NonUSSplashPage on:remove-extension={() => {
+    store.uninstallRally();
```

Comment:
```suggestion
    store.updatePlatformEnrollment(false);
```

Can you kindly check if this works?